src/scripts/ordo_subsets.py

```diff
@@ -0,0 +1,83 @@
+"""Creates a ROBOT template to update mondo terms with the correct ORDO subset annotation."""
+from argparse import ArgumentParser
+
+import pandas as pd
+
+from utils import remove_angle_brackets
+
+
+SUBSET_MAP = {
+    'Orphanet:557493': 'http://purl.obolibrary.org/obo/mondo#ordo_disease',  # disorder
+    'Orphanet:557492': 'http://purl.obolibrary.org/obo/mondo#ordo_group_of_disorders',  # group of disorders
+    'Orphanet:557494': 'http://purl.obolibrary.org/obo/mondo#ordo_clinical_subtype',  # subtype of a disorder
+}
+ROBOT_TEMPLATE_HEADER = {
+    'mondo_id': 'ID',
+    'ordo_id': 'A oboInOwl:hasDbXref',
+    'subset': 'A oboInOwl:inSubset',
+}
+
+
+def create_ordo_subsets_robot_template(
+    class_subsets_tsv_path: str, mondo_mappings_path: str, outpath: str,
+):
+    """Creates a ROBOT template to update mondo terms with the correct ORDO subset annotation."""
+    # Read: mondo.sssom.tsv
+    mondo_df = pd.read_csv(mondo_mappings_path, sep='\t', comment='#').rename(columns={
+        'subject_id': 'mondo_id',
+        'subject_label': 'mondo_label',
+    })
+    # - keep only exact matches
+    mondo_df = mondo_df[mondo_df['predicate_id'] == 'skos:exactMatch']
+
+    # Read: ORDO class subsets query results
+    df = pd.read_csv(class_subsets_tsv_path, sep='\t').rename(columns={
+        '?cls': 'ordo_id',
+        '?lbl': 'ordo_label',
+        '?grp': 'subset_ordo_class_id',
+        '?grp_lbl': 'subset_ordo_class_label',
+    }).sort_values(['ordo_id', 'ordo_label'])
+    # - remove duplicate rows where labels are a specific language variation, e.g. @en
+    #   this alternative works as of 2024/05/02, but is potentially trepidacious:
+    # df2 = df.drop_duplicates(subset=['ordo_id'], keep='last')
+    df['label_has_lang_variation'] = df['ordo_label'].map(lambda label: '@' in label)
```

Comment:
FYI: Deduped. We did a query of all subclasses of each subset, and we got some 6 duplicate entries with `@en` labels

Reply:
To clarify, the SPARQL results returned two rows where the IRI was the same and the labels varied by one including the `en` tag?

Reply:
Yikes! Can this be turned in an issue and assigned to me?

Reply:
Sure, https://github.com/monarch-initiative/mondo-ingest/issues/512